src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
+  }
+
+  md-tab-item {
+    padding-top: $toolbar-height-size-base / 2;
+    text-transform: none;
```

Comment:
Sure, I'm thinking the same. Looking at Ken's feedback felt like he wants view to be closer to mocks. That's why I've changed it.